zk_toolbox/crates/zk_supervisor/src/commands/test/revert.rs

```diff
@@ -22,12 +24,23 @@ pub fn run(shell: &Shell, args: RevertArgs) -> anyhow::Result<()> {
 
 fn install_and_build_dependencies(
     shell: &Shell,
+    args: &RevertArgs,
     ecosystem_config: &EcosystemConfig,
 ) -> anyhow::Result<()> {
     let _dir_guard = shell.push_dir(&ecosystem_config.link_to_code);
-    let spinner = Spinner::new("Installing and building dependencies...");
+    let spinner = Spinner::new(MSG_REVERT_TEST_INSTALLING_DEPENDENCIES);
     Cmd::new(cmd!(shell, "yarn install")).run()?;
     Cmd::new(cmd!(shell, "yarn utils build")).run()?;
+
+    // Build external node if necessary
+    if args.external_node {
+        Cmd::new(cmd!(
+            shell,
+            "cargo build --release --bin zksync_external_node"
+        ))
+        .run()?;
+    }
```

Comment:
why do we need it?

Reply:
we only need to run, not build

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2408/commits/50abe0c6bec18b21b3f714e3c98dd082150ce3b3